python/ray/metric.py

```diff
@@ -0,0 +1,7 @@
+from ray._raylet import (
+    Count,
+    Histogram,
+    Gauge,
+    Sum,
+    TagKey,
```

Comment:
We don't need to expose TagKey right? All tags seem to be passed by just a dict.